### PR TITLE
Buy improvements

### PR DESCRIPTION
**Pull request by AlexanderKolberg**

### CODE REVIEW

File: packages/sdk/src/react/ui/modals/_internal/components/quantityInput/index.tsx
Comment (w1that) [inline packages/sdk/src/react/ui/modals/_internal/components/quantityInput/index.tsx:1]: Making request to GenerateOfferTransaction with a quantity with decimals throws an error with cause: `"sequence marketplace generate offer transaction: create offer newValidatedCreateRequest: parse quantity: invalid number 1.5"`

File: packages/sdk/src/react/ui/modals/_internal/components/quantityInput/index.tsx [hunk 1]
```diff
@@ -160,10 +95,11 @@ export default function QuantityInput({
 					</Box>
 				}
 				numeric={true}
-				value={$quantity.get()}
+				value={quantity}
 				onChange={(e) => handleChangeQuantity(e.target.value)}
 				width={'full'}
 			/>
+			<Text color="negative">{invalidQuantity ? 'Invalid quantity' : ' '}</Text>
```
Comment (w1that) [inline packages/sdk/src/react/ui/modals/_internal/components/quantityInput/index.tsx:102]: fontsize

Can we make their fontsizes match?

Comment (AlexanderKolberg) [reply packages/sdk/src/react/ui/modals/_internal/components/quantityInput/index.tsx]: Thanks, done

**Review by w1that — APPROVED** [review]
Great!

**Pull request merged**